pyscf/tools/trexio.py

```diff
@@ -91,9 +93,10 @@ def _cc_to_trexio(cc_obj, trexio_file):
 def _mcscf_to_trexio(cas_obj, trexio_file):
     raise NotImplementedError
 
+
 def mol_from_trexio(filename, backend='h5'):
```

Comment:
`backend='h5'` can be removed here

Reply:
We now use `TREXIO_AUTO` in the reading functions, that's why the `backend` option does not need to be provided here